File: client/src/app/_models/process/candidateFlightItem.ts
export interface ICandiateFlightItem
{
    id: number;
    depId: number;
    depItemId: number;
    cvRefId: number;
    applicationNo: number;
    candidateName: string;
    categoryName: string;
    customerName: string;
    customerCity: string;
}

export class CandidateFlightItem implements ICandiateFlightItem
{
    id = 0;
    depId = 0;
    depItemId=0;
    cvRefId=0;
    applicationNo=0;
    candidateName='';
    customerName='';
    categoryName='';
    customerCity='';

}

File: api/Entities/Admin/CandidateFlightItem.cs
namespace api.Entities.Admin
{
    public class CandidateFlightItem: BaseEntity
    {
        public int CandidateFlightGrpId { get; set; }
        public int DepId { get; set; }
        public int CvRefId { get; set; }
        public int DepItemId { get; set; }
        public int ApplicationNo { get; set; }
        public string CategoryName { get; set; }
        public string CandidateName { get; set; }
        public string CustomerName { get; set; }
        public string CustomerCity { get; set; }
        
    }
}